src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountController.java

```diff
@@ -338,13 +338,25 @@ public ResponseEntity<? extends PiUser> createSystemAdminAccount(
         return ResponseEntity.ok(systemAdminAccountService.addSystemAdminAccount(account, issuerId));
     }
 
+    @ApiResponses({
+        @ApiResponse(responseCode = OK_CODE, description = "P&I Azure User Information"),
+        @ApiResponse(responseCode = AUTH_ERROR_CODE, description = NOT_AUTHORIZED_MESSAGE),
+        @ApiResponse(responseCode = NOT_FOUND_ERROR_CODE, description = "User not found")
+    })
+    @Operation(summary = "Get P&I Azure User info")
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping("/get-info/{provenanceUserId}")
+    public ResponseEntity<AzureAccount> getUserInfo(@PathVariable String provenanceUserId) {
+        return ResponseEntity.ok(accountService.retrieveUser(provenanceUserId));
+    }
+
     @ApiResponses({
         @ApiResponse(responseCode = OK_CODE, description = PI_USER),
         @ApiResponse(responseCode = AUTH_ERROR_CODE, description = NOT_AUTHORIZED_MESSAGE),
-        @ApiResponse(responseCode = NOT_FOUND_ERROR_CODE, description = "No user found with the "
-            + "email: {email} and provenance {provenance}")
+        @ApiResponse(responseCode = NOT_FOUND_ERROR_CODE, description = "No user found with the ")
     })
-    @Operation(summary = "Get an Admin user (excluding system admin) based on their email and provenance")
+    @Operation(summary = "Get an Admin user (excluding system admin) based on their email and provenance"
+        + "email: {email} and provenance {provenance}")
```

Comment:
Looks like a merge conflict here. Line 359 should be against the NOT_FOUND_ERROR_CODE

Reply:
yes, you are right. It's fixed.